node/get_status_node.go

```diff
@@ -220,7 +220,7 @@ func (n *StatusNode) StartLocalBackup() error {
 		} else {
 			backupDir = filepath.Join(n.config.RootDataDir, "backups")
 		}
-		fullPath := filepath.Join(backupDir, fmt.Sprintf("%x_user_data.bkp", accountIdentifier[:4]))
+		fullPath := filepath.Join(backupDir, fmt.Sprintf("%x_user_data.bkp", accountIdentifier[len(accountIdentifier)-4:]))
```

Comment:
In my opinion, the best identifier to use here is the profile key-uid. That aligns with other files/locations we store for the logged-in profile.

You can easily expose `func (db *Database) GetProfileKeypair() (*Keypair, error)` via `accounts` (or `wallet`) service like:
```
func (s *Service) GetProfileKeypair() (*accounts.Keypair, error) {

	return s.db.GetProfileKeypair()
}
``` 
and use it here:
```suggestion
		profileKeypair, err := n.accountsSrvc.GetProfileKeypair()
		if err != nil {
			return "", err
		}
		fullPath := filepath.Join(backupDir, fmt.Sprintf("%x_user_data.bkp", profileKeypair.KeyUID()))
```

Reply:
@saledjenic done. I also added a second commit because I remembered that using the data dir on mobile doesn't work